owenhallgren/stretch-project
# Feature/language icons

## What does this PR do?

- Adds dynamic rendering of images to match the language displayed on each Card

## How should it be tested in the UI?

- Run ```npm start``` and ensure all cards display the relevant image next to the language
